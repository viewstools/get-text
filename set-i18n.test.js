const setI18n = require('./set-i18n.js')

const READ_VIEW = `Read Vertical
Text
color #323232
fontFamily SlatePro
fontSize 20
fontWeight 600
text Assignment:

Name Text
text <name sets a default value

CaptureText
placeholder hi`

const READ_VIEW_TRANSLATIONS = {
  es: {
    'Read/Name/text <name gets a default value': 'tiene un valor por defecto',
    'Read/Text/text Assignment:': 'Tarea:',
    // TODO capture
  },
  // TODO test enIE
}

const READ_VIEW_THAT_ALREADY_HAS_TRANSLATIONS = `Read Vertical
Text
color #323232
fontFamily SlatePro
fontSize 20
fontWeight 600
text Assignment:
when <es
text Tarea:`

test('#setI18n', () => {
  expect(
    setI18n({
      source: READ_VIEW,
      translations: READ_VIEW_TRANSLATIONS,
      view: 'Read',
    })
  ).toMatchSnapshot()

  // expect(
  //   setI18n('Read', READ_VIEW_THAT_ALREADY_HAS_TRANSLATIONS)
  // ).toMatchSnapshot()
})
