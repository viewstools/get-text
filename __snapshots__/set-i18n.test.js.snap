// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#setI18n 1`] = `
"Read Vertical
Text
color #323232
fontFamily SlatePro
fontSize 20
fontWeight 600
text Assignment:

Name Text
text <name sets a default value

CaptureText
placeholder hi"
`;
